templates/tapestry.js

```diff
@@ -541,7 +541,6 @@ function tapestryTool(config){
                             url: apiUrl + "/tapestries/" + config.wpPostId + "/nodes/" + nodeId,
                             method: API_DELETE_METHOD,
                             success: function() {
-                                deleteLink(tapestry.dataset.links[linkToBeDeleted].source.id, tapestry.dataset.links[linkToBeDeleted].target.id, true, spliceIndex);
```

Comment:
I see there is a for loop here that is going over the links. Is this loop still necessary??

Reply:
I am not sure what `loop` you are referring to here. Do you mean the `for loop` in the `deleteLink()`?